src/NServiceBus.AcceptanceTesting/Support/IEndpointRunner.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace NServiceBus.AcceptanceTesting.Support
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface IEndpointRunner
+    {
+        bool FailOnErrorMessage { get; }
+        Task Start(CancellationToken token);
+        Task Whens(CancellationToken token);
```

Comment:
I don't like the `When` here. It makes sense on the endpoint level, but not for custom components. Wouldn't make a `OnStarted` make more sense?